### PR TITLE
Tn add rules

### PR DESCRIPTION
**Pull request by yzhang123**

### CODE REVIEW

File: nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify.py [hunk 1]
```diff
@@ -164,8 +164,9 @@ def __init__(
             v_date_graph = vDateFst(ordinal=v_ordinal_graph, deterministic=deterministic).fst
             time_final = pynini.compose(time_graph, v_time_graph)
             date_final = pynini.compose(date_graph, v_date_graph)
+            ordinal_final = pynini.compose(ordinal_graph, v_ordinal_graph.fst)
             range_graph = RangeFst(
-                time=time_final, date=date_final, cardinal=cardinal, deterministic=deterministic
+                time=time_final, date=date_final, cardinal=cardinal, ordinal=ordinal_final, deterministic=deterministic
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify.py:169]: same changes for audio-based normalization are needed. Note, CI doesn't run audio-based pytests.

File: nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify.py [hunk 2]
```diff
@@ -164,6 +164,7 @@ def __init__(
             v_date_graph = vDateFst(ordinal=v_ordinal_graph, deterministic=deterministic).fst
             time_final = pynini.compose(time_graph, v_time_graph)
             date_final = pynini.compose(date_graph, v_date_graph)
+            ordinal_final = pynini.compose(ordinal_graph, v_ordinal_graph.fst)
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify.py]: is this one used anywhere?

File: nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_with_audio.py [hunk 3]
```diff
@@ -154,6 +154,7 @@ def __init__(
             det_v_date_graph = vDate(ordinal=vOrdinal(deterministic=True), deterministic=True).fst
             time_final = pynini.compose(time_graph, det_v_time_graph)
             date_final = pynini.compose(date_graph, det_v_date_graph)
+            ordinal_final = pynini.compose(ordinal_graph, v_ordinal_graph)
```
Comment (ekmb) [inline nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_with_audio.py]: is this used?

Comment (ekmb) [reply nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_with_audio.py]: why did you remove date range changes?

**Review by ekmb — APPROVED** [review]

File: nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_with_audio.py (hunk 3 above)
Comment (yzhang123) [reply nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify_with_audio.py]: the non deterministic graph doesn't build unfortunately, its too large do to difference and composition with date.

**Pull request merged**